tools/decode-test-data.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+PARSE_PAYLOAD_SCRIPT=$(dirname "${0}")/parse-payload.sh
+
+# Parse test data payloads into a directory out of the source tree for editing.  These payloads can then
+# be written back to the tree (along with any schema updates) using the encode-test-data.sh script.
+
+if [[ "$#" -ne 2 ]]; then
+  echo "Usage: $0 <path_to_src_test_data_dir> <path_to_dest_test_data_dir>" >&2
+  exit 1
+fi
+
+if [[ ! -d "$1" ]]; then
+  echo "Source directory '$1' does not exist."
+  exit 2
+fi
+
+if [[ ! -d "$2" ]]; then
+  echo "Destination directory '$2' does not exist."
+  exit 3
+fi
+
+for subdir in ${1}/*; do
+  TEST_CFG=$(basename "${subdir}")
+  SRC_ENV_DIR="${subdir}/v0/environment"
+
+  if [[ ! -d "${SRC_ENV_DIR}" ]]; then
+    continue
```

Comment:
just double checking... there's never an instance where there's no environment config, but there's a landing zone config, right?

Reply:
Not for a working test configuration, so it would be OK if this script missed it.